### PR TITLE
Update `pt_BR` localized messages

### PR DESCRIPTION
**Pull request by Gudahtt**
These translations were provided by Lionbridge.

### CODE REVIEW

**Comment by Gudahtt** [comment]
This PR has been rebased onto `develop` to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:65]: In this case, I believe we should use the plural of `endereço`, `endereços`. Because we have a catalogue that stores multiple addresses :)

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:68]: The correct abbreviation of `por exemplo` would be `p ex` (corresponding to e.g.). But it is common to see different usages (`Exemplo:`, `Ex.,`, etc.).

Source (pt-br dictionary): https://michaelis.uol.com.br/moderno-portugues/como-consultar/abreviaturas/

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:77]: Without more context, it is hard to know if the male or female version of the noun should be used.

e.g. when talking about different "levels", it would make more sense to use the male noun:
```
Basic Level - Nível Básico
Advanced Level - Nível Avançado
```

But when talking about options (e.g. Advanced options) the correct usage would be the female noun:
```
Advanced Options - Opçōes Avançadas
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:83]: This sounds a bit "google-translated-ish". I can't really explain why. But here are a few changes I'd make:

1. Change `Acessar recursos DO desenvolvedor` with `Acessar recursos DE desenvolvedor`.
2. Change `Baixar logs de estado` with `Baixar logs`
3. Change `Configurar testnets e RPC personalizada` with `Configurar testnets e RPC personalizadaS`

**Comment by Gudahtt** [comment]
This has been rebased onto `develop`, and one unused message was removed (`swapGetQuotes`).

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:65]: Thanks for the suggestion! I will apply this change:
```suggestion
    "message": "Adicionar ao catálogo de endereços"
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:68]: Thanks for the suggestion! I'll update this to `p ex` then.

I'm a little unclear on the punctuation :thinking: Does this look correct? Or should I remove the comma, or the periods?
```suggestion
    "message": "p. ex., John D."
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:77]: The context here is "Advanced Options", so the female noun would be correct I guess!

We use this message in a few different places, but it always means "advanced" as in "advanced options".

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:83]: Thanks! Does this look OK?

```suggestion
    "message": "Acessar recursos de desenvolvedor, baixar logs, redefinir conta, configurar testnets e RPC personalizadas"
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [reply app/_locales/pt_BR/messages.json:68]: I did a quick search (https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/ponto-no-final-da-frase-apos-ponto-da-abreviatura/31884) and it looks like the correct usage would be `p.ex.: John D.`.

I can't say if this is really the right way of using it, but I have definitely seen `p. ex.:` being commonly used.

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [reply app/_locales/pt_BR/messages.json:83]: Yes. It reads better :)

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:68]: Ok, well in that case I'll go with what you found:

```suggestion
    "message": "p.ex.: John D."
```

We can always change it later!

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:95]: I believe the verb tense here is wrong. In the original English message, we have `Browsing a website`, using present continuous (the action is happening atm).

In this case, I believe the right tense for 'Navegar' would be 'Navegando'. So we should have:
```
Navegando por um website com uma conta não conectada selecionada
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:117]: I believe the intention was to spend up to a value.

I'd change the translation to:
```
Permitir que $1 saque e gaste um valor máximo de:
```

Or maybe even simpler:
```
Permitir que $1 saque e gaste no máximo:
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:131]: I think the `do` is unnecessary.

This reads better:

```
Uma carteira Ethereum no seu navegador
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:164]: This isn't wrong, it just doesn't read naturally. I can't explain why.

Change the sentence to:

```
Se a tentativa de cancelamento tiver sucesso, a taxa de transação acima será cobrada.
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:167]: I haven't seen anyone referring to blockchain as a male noun in pt-br. I'd use it as a female noun (or undefined):

```
Tentando conexão com a blockchain
```

Or

```
Tentando conexão com blockchain
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:179]: You may contract `de o` into `do`

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:222]: When using `built` referring to software development, I believe we use `desenvolvido` instead of `construido`.

Construido is more when we are refering to a building or some object that was built.

I'd go with:
```
O MetaMast é projetado e desenvolvido na Califórnia
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [inline app/_locales/pt_BR/messages.json:240]: This is a bit weird to read. I'd say that either we specify the unit between parentheses, or we leave it to the end of the sentence.

```
Taxa (em gas) de cancelamento
```

Or 

```
Taxa de cancelamento em gas
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:95]: Thanks! I'll apply that suggestion:
```suggestion
    "message": "Navegando por um website com uma conta não conectada selecionada"
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:117]: Good catch! That does seem more correct. I'll apply the simpler suggestion:
```suggestion
    "message": "Permitir que $1 saque e gaste no máximo:",
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:131]: :+1: 
```suggestion
    "message": "Uma carteira Ethereum no seu navegador",
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:164]: :+1: 
```suggestion
    "message": "Enviar esta tentativa não garante que a transação original será cancelada. Se a tentativa de cancelamento tiver sucesso, a taxa de transação acima será cobrada."
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:167]: Interesting :thinking: I'll take the second suggestion for now. 
```suggestion
    "message": "Tentando conexão com blockchain."
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:179]: :+1: 
```suggestion
    "message": "Defina o tempo ocioso, em minutos, antes do MetaMask ser bloqueado."
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:222]: Makes sense!
```suggestion
    "message": "O MetaMask é projetado e desenvolvido na Califórnia."
```

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [reply app/_locales/pt_BR/messages.json:240]: Thanks! I'll take the second suggestion for now.

```suggestion
    "message": "Taxa de cancelamento em gas"
```

File: app/_locales/pt_BR/messages.json
Comment (lucassaldanha) [reply app/_locales/pt_BR/messages.json]: Yeah, this gender-based noum thing in pt-br can be annoying. And there isn't a rule or anything to define it... And for words that aren't really translated it is even weirder.

Some other examples:
Internet -> we don't translate the word, but it is used as a female noun.
Computer -> Computador (male noun)

So the sentence: `The computer is connected to the internet` becomes `O computador está conectado à internet`.

:)

**Comment by Gudahtt** [comment]
I've just rebased this again to resolve conflicts (two messages were removed)

**Comment by Gudahtt** [comment]
I've just rebased this again to resolve conflicts (a few messages were removed)

**Comment by Gudahtt** [comment]
Rebased to resolve conflicts

**Comment by mcmire** [comment]
Probably should wait for #13206 to be merged before merging this (if we still want to do this).

**Comment by mcmire** [comment]
Rebased against `develop`. #13206 went in which filled in a lot of missing translations. That left only a handful that were listed in the English translation but not in this one in particular.

**Review by brad-decker — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]

File: app/_locales/pt_BR/messages.json
Comment (Gudahtt) [inline app/_locales/pt_BR/messages.json:1290]: This looks like a mistake

**Comment by brad-decker** [comment]
@Gudahtt should we just close this and rely on the new translation service? I'm unsure of where we stand with that integration and if it would pick these up

**Review by brad-decker — CHANGES_REQUESTED** [review]
blocking merge due to @Gudahtt 's concerns

**Comment by brad-decker** [comment]
These transalations should be covered by our new provider. Closing this and letting the new service handle.

**Pull request closed without merging**
